corehq/apps/repeaters/tests/test_repeater.py

```diff
@@ -302,6 +296,7 @@ def test_excluded_case_types_are_not_forwarded(self):
         CaseFactory(self.domain_name).post_case_blocks([non_white_listed_case])
         self.assertEqual(1, len(self.repeat_records(self.domain_name).all()))
 
+    # @run_with_all_backends
```

Comment:
(I don't think they could be using any app builder feature including case management or save to case in form builder and get into a state where that wasn't true)